server/src/main/java/org/apache/uniffle/server/ShuffleServer.java

```diff
@@ -318,19 +318,16 @@ private void initialization() throws Exception {
             shuffleMergeManager);
     shuffleTaskManager.start();
     ShuffleServerMetrics.addLabeledGauge(
-        USED_DIRECT_MEMORY_SIZE_BY_NETTY, () -> (double) PlatformDependent.usedDirectMemory());
+        USED_DIRECT_MEMORY_SIZE_BY_NETTY, PlatformDependent::usedDirectMemory);
     ShuffleServerMetrics.addLabeledGauge(
         USED_DIRECT_MEMORY_SIZE_BY_GRPC_NETTY,
-        () ->
-            (double)
-                io.grpc.netty.shaded.io.netty.util.internal.PlatformDependent.usedDirectMemory());
+        io.grpc.netty.shaded.io.netty.util.internal.PlatformDependent::usedDirectMemory);
```

Comment:
Could we remove the package name?

Reply:
I'm afraid not, since there are two `PlatformDependent` used in these file.